### ISSUE TITLE
Add admin page to easily delete pedophile pastes

### ISSUE THREAD

**Issue by sametmax** [Tygs/0bin#87]
Settings can _optionally_ contain: 

```
ADMIN_SECRET = {
    "url": "hjkl789HJK",
    "password": "wololo6789HJK#"
}
```

If it does, the site declare a new secret URL: "/admin/hjkl789HJK"

When you log into it using the password, you get a form in which you can enter a full 0bin URL or just a URL ID, which will delete the paste.

**Comment by Shir0kamii** [comment]
Alright, I'll try to work on this. The design might not be elegant, but I will try to implement the feature.

**Comment by ksamuel** [comment]
Implemented in V1.0.0
